src/Twitter.php

```diff
@@ -803,14 +804,21 @@ public function friendshipsCreate($id, array $params = []) : Response
      *
      * Returns the next cursor if there are more to be returned.
      *
-     * @param int|string $id
+     * $id may be one of any of the following:
+     *
+     * - a single user identifier
+     * - a single screen name
+     * - a list of user identifiers
```

Comment:
This should be split into a singular and a plural method (new API).
Widening the accepted types is also a BC break, as it breaks LSP compliance for subclasses.